nucypher/network/server.py

```diff
@@ -318,6 +330,11 @@ def receive_treasure_map(self, treasure_map_id, request: Request):
             self.log.info("Bad TreasureMap ID; not storing {}".format(treasure_map_id))
             assert False
 
+    def status(self, request: Request):
+        headers = {"Content-Type": "text/html", "charset":"utf-8"}
+        content = self._status_template.render(known_nodes=self._node_tracker.values())
```

Comment:
Why don't I see the name of my node there? (while `status` in console shows it)

Reply:
Although, could be a problem on my side (e.g. not necessarily the port is truly open)

Reply:
Upd: actually no, status doesn't show it, but the teacher seems to know me

Reply:
I'm not sure I understand the question - are you just wondering how / why the node names are shown in the template without being passed individually?

The template iterates through the nodes.

Reply:
The question is: when you iterate, is your own node in the list?

Reply:
No - This iterates over `known_nodes`

Reply:
For me, looks like it's not, but I am not sure if it's a bug or feature. The learningloop tells me that other nodes seemingly see my node (judging by the number of nodes).

Reply:
Heh - sounds like you have an unexpected number of nodes?

Reply:
Ah, I see. I think, makes sense to include the "self" node in the list also (maybe on the top). But yeah, that's a feature than

Reply:
Yeah - I agree that it's useful to have the self-Ursula on the page somewhere conspicuous

Reply:
https://github.com/nucypher/nucypher/issues/528